cocos/particle/renderer/particle-system-renderer-data.ts

```diff
@@ -170,6 +170,7 @@ export default class ParticleSystemRenderer {
         } else {
             const effectName = val.effectName;
             if (effectName.indexOf('particle') === -1 || effectName.indexOf('particle-gpu') !== -1) {
+                console.log('Should use particle effect!');
```

Comment:
please use warnID instead, it will cause a failure on user action, so user should notice that, and we should put all message into EngineErrorMap.md